lib/open_project/files.rb

```diff
@@ -56,5 +56,14 @@ def create_uploaded_file(name: 'test.txt',
       tmp = create_temp_file name: name, content: content, binary: binary
       Rack::Multipart::UploadedFile.new tmp.path, content_type, binary
     end
+
+    def mock_uploaded_file(name: 'test.txt',
+                           content_type: 'text/plain',
+                           content: 'test content',
+                           binary: false)
+
+      tmp = create_temp_file name: name, content: content, binary: binary
+      Rack::Test::UploadedFile.new tmp.path, content_type, binary
+    end
```

Comment:
I think there is a better place for code you require only in your tests than in the production code.